<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addmoredetails</class>
 <widget class="QDialog" name="addmoredetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>677</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#addmoredetails{
	border-image: url(:/images/Myresources/background.png);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="1" rowspan="2" colspan="2">
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">#frame{
	border: 2px;
	border-radius: 10px;
	background-color: rgba(30, 30, 30, 150);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1" rowspan="2" colspan="2">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>Berlin Sans FB</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(170, 255, 255);</string>
          </property>
          <property name="text">
           <string>Add More Details</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>5</number>
            </property>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <family>Berlin Sans FB</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Contact Info</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <family>Berlin Sans FB</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">*{
	color: rgb(170, 255, 255);
}</string>
              </property>
              <property name="text">
               <string>Blood Group</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="font">
               <font>
                <family>Berlin Sans FB</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">*{
	color: rgb(170, 255, 255);
}</string>
              </property>
              <property name="text">
               <string>Gender</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>5</number>
            </property>
            <item>
             <widget class="QLineEdit" name="lineEdit_5">
              <property name="font">
               <font>
                <family>Berlin Sans FB</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">#lineEdit_5{
background : transparent;
border: none;
border-bottom: 1px solid ;
	color: rgb(30, 30, 30);
}
#lineEdit_5 :hover {
    border-bottom: 2px solid #00BFFF;   /* hover color */
}

#lineEdit_5 :focus {
    border-bottom: 2px solid #0078FF;   /* focus color */
}
</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_3">
              <property name="font">
               <font>
                <family>Berlin Sans FB</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">#lineEdit_3{
background : transparent;
border: none;
border-bottom: 1px solid ;
	color: rgb(30, 30, 30);
}
#lineEdit_3 :hover {
    border-bottom: 2px solid #00BFFF;   /* hover color */
}

#lineEdit_3 :focus {
    border-bottom: 2px solid #0078FF;   /* focus color */
}
</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_4">
              <property name="font">
               <font>
                <family>Berlin Sans FB</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">#lineEdit_4{
background : transparent;
border: none;
border-bottom: 1px solid ;
	color: rgb(30, 30, 30);
}
#lineEdit_4 :hover {
    border-bottom: 2px solid #00BFFF;   /* hover color */
}

#lineEdit_4 :focus {
    border-bottom: 2px solid #0078FF;   /* focus color */
}
</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>Berlin Sans FB</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">*{
color: rgb(170, 255, 255);
}</string>
            </property>
            <property name="text">
             <string>Weight</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox">
            <property name="font">
             <font>
              <family>Berlin Sans FB</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">#comboBox{
background-color: transparent;
    border: none;
	color: rgb(170, 255, 255);          /* Change text color if needed */
}

comboBox::drop-down {
    background: transparent;
	color: rgb(170, 255, 255);
    border: none;
}

comboBox QAbstractItemView {
    background: rgba(0, 0, 0, 100);   /* popup list transparency */
    color: white;
}</string>
            </property>
            <item>
             <property name="text">
              <string>in kg</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>in lb</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="font">
             <font>
              <family>Berlin Sans FB</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">#lineEdit{
background : transparent;
border: none;
border-bottom: 1px solid ;
	color: rgb(30, 30, 30);
}
#lineEdit :hover {
    border-bottom: 2px solid #00BFFF;   /* hover color */
}

#lineEdit :focus {
    border-bottom: 2px solid #0078FF;   /* focus color */
}
</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>Berlin Sans FB</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">*{
color: rgb(170, 255, 255);}
</string>
            </property>
            <property name="text">
             <string>Height</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_2">
            <property name="font">
             <font>
              <family>Berlin Sans FB</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">#comboBox_2{
background-color: transparent;
    border: none;
color: rgb(170, 255, 255);         /* Change text color if needed */
}

comboBox_2::drop-down {
    background: transparent;
    border: none;
}

comboBox_2 QAbstractItemView {
    background: rgba(0, 0, 0, 100);   /* popup list transparency */
    color: white;
}</string>
            </property>
            <item>
             <property name="text">
              <string>in feet</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>in cm</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_2">
            <property name="font">
             <font>
              <family>Berlin Sans FB</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">#lineEdit_2{
background : transparent;
border: none;
border-bottom: 1px solid ;
	color: rgb(30, 30, 30);
}
#lineEdit_2 :hover {
    border-bottom: 2px solid #00BFFF;   /* hover color */
}

#lineEdit_2 :focus {
    border-bottom: 2px solid #0078FF;   /* focus color */
}
</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="font">
             <font>
              <family>Berlin Sans FB</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">#pushButton_2{
color :rgb(0, 255, 255);
background-color: transparent;
  border: none;
}</string>
            </property>
            <property name="text">
             <string>do it later</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="font">
             <font>
              <family>Berlin Sans FB</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">#pushButton{
color :rgb(0, 255, 255);
background-color: transparent;
  border: none;
}</string>
            </property>
            <property name="text">
             <string>save</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="1" colspan="2">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>238</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="3" rowspan="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>177</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="1" colspan="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>238</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" rowspan="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>39</width>
          <height>177</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="3" rowspan="2">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>103</width>
       <height>293</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1" colspan="2">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>353</width>
       <height>172</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" rowspan="2">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>104</width>
       <height>293</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" colspan="2">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>353</width>
       <height>172</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
